packages/eslint-plugin/docs/rules/no-unnecessary-boolean-literal-compare.mdx

```diff
@@ -101,11 +101,11 @@ Examples of code for this rule with `{ allowComparingNullableBooleansToFalse: fa
 
 ```ts option='{ "allowComparingNullableBooleansToFalse": false }'
 declare const someUndefinedCondition: boolean | undefined;
-if (someUndefinedCondition === false) {
```

Comment:
(optional)

Question - is there any reason for these to negate the "Incorrect" code rather than the "Correct" code here? It feels weird to me to have `!inputExpression` => `!outputExpression` when `inputExpression` => `outputExpression` would be much simpler. (_This isn't a loaded question; I just genuinely can't think of a reason. I note that the chart below does it that way, too, which you've probably followed for consistency, but... should we just change it there too?_)

Either way, though, your change fixes it from incorrect to correct, following the existing pattern, so it's totally optional if you even want to think about this comment or not! 🙂

Reply:
Great, go for it! 🚀🚀

Reply:
Done! 😄